apps/docs-app/src/pages/index.module.css

```
@@ -20,6 +20,16 @@
   display: flex;
   align-items: center;
   justify-content: center;
+  flex-wrap: wrap;
+  gap: 1rem;
```

praise: Great solution!

---

issue: Use the Infima spacing variable instead of hard-coding `1rem`.
```suggestion
  gap: var(--ifm-global-spacing);
```